src/index.ts

```diff
@@ -9,3 +9,4 @@
  */
 
 export { format } from './utils/utils';
+export * from './components';
```

Comment:
Is it possible to explicitly export the `.d.ts` file like:
```suggestion
export type * from './components.d.ts';
```

It would make it more explicit what is being exported as I got confused by this. For example, if someone would create the following files:

- `src/components.js`
- `src/components/index.js`

The exports of these files would be preferred, right?

Reply:
Yeah - good point. Tested locally that this works - let's go with this suggestion